dialogflow-cx/vpc-sc-demo/frontend/src/PrivacyPolicy.js

```diff
@@ -0,0 +1,107 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+import Link from '@mui/material/Link';
+import Typography from '@mui/material/Typography';
+
+function PrivacyPolicyPage(props) {
+  const openSourceAndPubliclyViewable = (
+    <Link
+      target="_blank"
+      href="https://developers.google.com/identity/protocols/oauth2"
+      variant="body1"
+    >
+      Open Source and publicly viewable
+    </Link>
+  );
+  const AccessToken = (
+    <Link
+      target="_blank"
+      href="https://github.com/GoogleCloudPlatform/contact-center-ai-samples/tree/main/dialogflow-cx/vpc-sc-demo"
+      variant="body1"
+    >
+      Access Token
+    </Link>
+  );
+
+  const launchPadLink = (
+    <Link
+      style={{cursor: 'pointer'}}
+      onClick={() => {
+        props.dataModel.activePage.set('liveDemo');
+      }}
+    >
+      Live Demo &quot;Launch Pad&quot;
+    </Link>
+  );
+
+  return (
+    <>
+      <Typography variant="h4" sx={{my: 3}} id="privacyPolicy">
+        Privacy Policy
+      </Typography>
+      <Typography paragraph>
+        This demo app is built and maintained by the Developer Enablement
+        Engineering Team at Google Cloud, but is not an official Google Project.
+        When you use this service, you are entrusting us with you data, and we
+        take that responsibility very seriously. While this site does collect
+        several pieces of data as you explore the site via cookies, no
+        personally-identifying data is collected or stored. There is an option
+        to &quot;Log In&quot; with your Google identity using OAuth to access
+        the advanced features of the demo, but this additional data is only
+        stored for a short period of time. All the source code for this website
```

Comment:
Maybe add a `Data Storage` section below that details what auth/session data is stored, where it is stored, and for how long until it is deleted (i.e., the auth/session data server side, as opposed to the cookie). This would help users understand the context of their session/auth data and how it's only used during the lifecycle of provisioning cloud resources while using the demo app.

Reply:
Done.